tests/app/clients/test_twilio.py

```diff
@@ -84,16 +91,71 @@ def test_send_sms_calls_twilio_correctly(notify_api, mocker):
     assert d["Body"] == "my message"
 
 
+@pytest.mark.parametrize('sms_sender_id', ['test_sender_id', None])
```

Comment:
moving forward, for new tests that use parameterize, can we please start using IDs to identify the test cases? ids=["has sender id", "has no sender id"] (or whatever is under test here). This is a problem throughout all test cases, but it would be nice to start somewhere.

info: https://docs.pytest.org/en/6.2.x/example/parametrize.html#different-options-for-test-ids

Reply:
This is a great idea! You should bring it up at the next engineering meeting on Friday. I can start using ids on the tests I've added here and we can get started on that.